packages/dapp-svelte-wallet/react-ui/src/store.js

```diff
@@ -0,0 +1,15 @@
+// The code in this file requires an understanding of Autodux.
```

Comment:
`autodux` was a necessary evil to get us out of the worse evil of all the duplication and interwoven code that using Redux had in the autoswap UI.  The watchword with Redux is "you'll know when you need it".

If you have good reasons for using it, then I can accept that.  Otherwise, I would prefer not to use it when we have this chance to start from scratch.